extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/multipart/MultipartSupport.java

```diff
@@ -0,0 +1,116 @@
+package io.quarkus.resteasy.reactive.server.runtime.multipart;
+
+import static org.jboss.resteasy.reactive.common.util.DeploymentUtils.loadClass;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Set;
+
+import javax.ws.rs.InternalServerErrorException;
+import javax.ws.rs.NotSupportedException;
+import javax.ws.rs.RuntimeType;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.ext.MessageBodyReader;
+
+import org.jboss.logging.Logger;
+import org.jboss.resteasy.reactive.common.util.types.TypeSignatureParser;
+import org.jboss.resteasy.reactive.server.core.ResteasyReactiveRequestContext;
+import org.jboss.resteasy.reactive.server.core.ServerSerialisers;
+import org.jboss.resteasy.reactive.server.handlers.RequestDeserializeHandler;
+import org.jboss.resteasy.reactive.server.spi.ServerMessageBodyReader;
+
+import io.vertx.ext.web.FileUpload;
+import io.vertx.ext.web.RoutingContext;
+
+/**
+ * This class isn't used directly, it is however used by generated code meant to deal with multipart forms.
+ */
+public final class MultipartSupport {
+
+    private static final Logger log = Logger.getLogger(RequestDeserializeHandler.class);
+
+    private static final Annotation[] EMPTY_ANNOTATIONS = new Annotation[0];
+
+    private MultipartSupport() {
+    }
+
+    public static Object convertFormAttribute(String value, String typeClassStr, String signature, String mediaTypeStr,
+            ResteasyReactiveRequestContext context) {
+        if (value == null) {
+            return null;
+        }
+
+        Class<?> type = loadClass(typeClassStr);
+        Type genericType = type;
+        if (signature != null) {
```

Comment:
Not a blocker, but shouldn't this be parsed in the static init block of the generated class and stored in a field, same with the media type?

Reply:
Good idea, I'll do that

Reply:
Done